stages/org.osbuild.systemd.unit

```diff
@@ -28,7 +28,7 @@ SCHEMA = r"""
 "properties": {
   "unit": {
     "type": "string",
-    "pattern": "^[\\w:.\\\\-]+[@]{0,1}[\\w:.\\\\-]*\\.service$"
+    "pattern": "^[\\w:.\\\\-]+[@]{0,1}[\\w:.\\\\-]*\\.(service|timer)$"
```

Comment:
Can you amend the commit message for this to match the conventions of the repo?

Something like:
```
stages/systemd: support timer dropins
```